js/control/TrackAnalysis.js

```diff
@@ -75,14 +75,14 @@ BR.TrackAnalysis = L.Class.extend({
     /**
      * Everytime the track changes this method is called:
      *
-     * - calculate statistics (way type, surface, smoothness)
+     * - calculate statistics (way type, max speed, surface, smoothness)
      *   for the whole track
      * - renders statistics tables
      * - create event listeners which allow to hover/click a
      *   table row for highlighting matching track segments
      *
      * @param {Polyline} polyline
-     * @param {Array} segments
+     * @param {Array} segments route segments between waypoints
```

Comment:
nit: `segments route segments`? Shouldn't it be `route segments` alone?

Reply:
The first `segments` refers to the variable name, the three following words are the description.

As per [JSDoc](https://jsdoc.app/tags-param) a hyphen should be inserted after the parameter name. I'll add this in a follow-up pull request.

Reply:
Ah yeah ok my bad, :+1: